apps/web/src/App.tsx

```diff
@@ -40,7 +40,28 @@ import LogRocket from 'logrocket';
 import setupLogRocketReact from 'logrocket-react';
 
 if (LOGROCKET_ID && window !== undefined) {
-  LogRocket.init(LOGROCKET_ID);
+  LogRocket.init(LOGROCKET_ID, {
+    release: 'v0.11.0',
```

Comment:
This shouldn't be hardcoded like this here. Should be extracted from the package.json or injected in any other dynamic way. (env variables for example) it can get out of sync quite fast